auth-web/src/components/auth/staff/DissolutionSchedule.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+  <div id="summary-define-company">
+    <section>
```

Comment:
This can be your top-level block. The div is useful if there are multiple children, but you only have 1.